packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -534,6 +534,46 @@ void main() {
         equals('TextInputAction.done'));
   });
 
+  testWidgets('connection is closed when TextInputClient.onConnectionClosed message received', (WidgetTester tester) async {
```

Comment:
Another interesting test would be to use the text field again after the connection is closed (it should create a new connection and work fine).

Reply:
+1

Reply:
Sure, it sounds very useful I'll add it.

Reply:
Done.